bn-data-access/src/base/baseLookupImpl.ts

```diff
@@ -0,0 +1,57 @@
+import { get, orderBy } from 'lodash';
+import { couchService } from '@boatnet/bn-couch';
+import { Client } from 'davenport';
+import { mongoService } from '@boatnet/bn-mongo';
+import { dbConfig } from '../config/config';
+import { Base } from './base';
+
+interface CouchInfo {
+    view: string,
+    label: string,
+    value: string,
+    displayCode?: string,
+    queryOptions?: any
+}
+
+interface MongoInfo {
```

Comment:
hopefully we can review during tomorrow's meeting - I'm not sure I understand what these fields are.

would this setup work if we stored the entire object in couch, but only a reference in mongo?  Guess we should talk about how/whether the service can support different model implementations - ie. storing a reference instead of the entire object.